e2e/callback_test.go

```diff
@@ -195,351 +196,322 @@ func (suite *CbTestSuite) SetupTest() {
 
-	// Send from ChainA to ChainC
+	// A -> C token_id 4
 	sendIcsFromChainAToC(suite, suite.cw721A.String(), "4", memo, true)
 
 	// Get the cw721 address on ChainC when received from ChainB
 	ACClassId := fmt.Sprintf("%s/%s/%s", suite.pathAC.EndpointB.ChannelConfig.PortID, suite.pathAC.EndpointB.ChannelID, suite.cw721A)
-	ACCw721 := queryGetNftForClass(suite.T(), suite.chainC, suite.bridgeC.String(), ACClassId)
+	ACCw721 := test_suite.QueryGetNftForClass(suite.T(), suite.chainC, suite.bridgeC.String(), ACClassId)
 
 	// Confirm tester is the owner on Chain C of the nft id "4"
-	chainCOwner = queryGetOwnerOf(suite.T(), suite.chainC, ACCw721, "4")
+	chainCOwner = test_suite.QueryGetOwnerOf(suite.T(), suite.chainC, ACCw721, "4")
 	require.Equal(suite.T(), chainCOwner, suite.testerC.String())
 
 	confirmNftContracts(suite.chainA, suite.chainC, suite.testerA.String(), suite.testerC.String(), suite.cw721A.String(), ACCw721)
 
 	// Let send back all NFTs to Chain A
+	// C -> A token_id 4
 	sendIcsFromChainCToA(suite, ACCw721, "4", memo, true)
 	confirmNftContracts(suite.chainC, suite.chainA, suite.testerC.String(), suite.testerA.String(), ACCw721, suite.cw721A.String())
 
+	// C -> B token_id 2
 	sendIcsFromChainCToB(suite, BCCw721, "2", memo, true)
 	confirmNftContracts(suite.chainC, suite.chainB, suite.testerC.String(), suite.testerB.String(), BCCw721, suite.cw721B.String())
-
+	// B -> A again token_id 2
 	sendIcsFromChainBToA(suite, suite.cw721B.String(), "2", memo, true)
 	confirmNftContracts(suite.chainB, suite.chainA, suite.testerB.String(), suite.testerA.String(), suite.cw721B.String(), suite.cw721A.String())
 
-	chainAOwner1 := queryGetOwnerOf(suite.T(), suite.chainA, suite.cw721A.String(), "2")
+	chainAOwner1 := test_suite.QueryGetOwnerOf(suite.T(), suite.chainA, suite.cw721A.String(), "2")
 	require.Equal(suite.T(), chainAOwner1, suite.testerA.String())
-	chainAOwner2 := queryGetOwnerOf(suite.T(), suite.chainA, suite.cw721A.String(), "4")
+	chainAOwner2 := test_suite.QueryGetOwnerOf(suite.T(), suite.chainA, suite.cw721A.String(), "4")
 	require.Equal(suite.T(), chainAOwner2, suite.testerA.String())
 
 	// NFTs should no exist on Chain B and Chain C, they should be burned and query for owner should error
-	err := queryGetOwnerOfErr(suite.T(), suite.chainB, suite.cw721B.String(), "2")
+	err := test_suite.QueryGetOwnerOfErr(suite.T(), suite.chainB, suite.cw721B.String(), "2")
 	require.Error(suite.T(), err)
-	err = queryGetOwnerOfErr(suite.T(), suite.chainC, BCCw721, "2")
+	err = test_suite.QueryGetOwnerOfErr(suite.T(), suite.chainC, BCCw721, "2")
 	require.Error(suite.T(), err)
-	err = queryGetOwnerOfErr(suite.T(), suite.chainC, ACCw721, "4")
+	err = test_suite.QueryGetOwnerOfErr(suite.T(), suite.chainC, ACCw721, "4")
 	require.Error(suite.T(), err)
 }
+
+func callbackMemo(srcCallback, srcReceiver, dstCallback, dstReceiver string) string {
```

Comment:
I'd prefer using same namings as it is used for props. like `srcCallback` -> `ack_callback_data`, etc.